src/openhab.h

```diff
@@ -99,18 +100,30 @@ class OpenHAB : public Integration {
     const QString* lookupComplexLightItem(const QString& entityId, LightDef::Attributes attr);
 
  private:
-    QNetworkAccessManager       _sseNetworkManager;
-    QNetworkReply*              _sseReply;
-    QTimer*                     _sseReconnectTimer;
-    QString                     _url;
-    QString                     _token;
-    QNetworkAccessManager       _nam;
-    QList<EntityInterface*>     _myEntities;     // Entities of this integration
-    QRegExp regex_colorvalue = QRegExp("[0-9]?[0-9]?[0-9][,][0-9]?[0-9]?[0-9][,][0-9]?[0-9][.]?"
-                                       "[0-9]?[0-9]?[0-9]?[0-9]?");
+    QNetworkInterface       _iface;
+    QNetworkAccessManager*  _sseNetworkManager;
+    QNetworkReply*          _sseReply;
+    QTimer*                 _sseReconnectTimer;
+    QString                 _url;
+    QString                 _token;
+    int                     _networktries = 0;
+    QNetworkAccessManager*  _nam;
+    bool                    _flagleaveStandby = false;
+    QList<EntityInterface*> _myEntities;  // Entities of this integration
+    QRegExp                 regex_colorvalue =
+        QRegExp("[0-9]?[0-9]?[0-9][,][0-9]?[0-9]?[0-9][,][0-9]?[0-9][.]?[0-9]?[0-9]?[0-9]?[0-9]?");
     QRegExp regex_brightnessvalue = QRegExp("[1]?[0-9]?[0-9]");
-    int                         _tries;
-    bool                        _flagStandby;
-    bool                        _flagOpenHabConnected = false;
-    QObject* context = new QObject(this);
+    int     _tries;
+    bool    _flagStandby;
+    // bool     _flagprocessitems = false;
+    bool _flagOpenHabConnected = false;
+    // QObject* context = new QObject(this);
+    QString        _tempJSONData = "";
+    OpenHAB*       context_openHab;
```

Comment:
Please stick to one naming convention: variable has no prefix as the others and no lowerCamelCase